packages/manager/modules/bm-server-components/src/general-information/installation/ovh/server-installation-ovh.controller.js

```diff
@@ -0,0 +1,2668 @@
+
+  validationRaid(partition) {
+    if (partition.fileSystem === this.$scope.constants.warningSwap) {
+      set(partition, 'raid', this.$scope.constants.warningRaid1);
+    }
+    if (
+      this.$scope.installation.nbDiskUse > 1 &&
+      !this.$scope.informations.raidController
+    ) {
+      this.$scope.warning.raid0 =
+        !this.$scope.errorInst.raid0 &&
+        partition.raid === this.$scope.constants.warningRaid0;
+    }
+    this.validationSize(partition);
+  }
+
+  // ------SIZE VALIDATION------
+
+  hasErrorSize() {
+    return (
+      this.$scope.errorInst.partitionSizeOver ||
+      this.$scope.errorInst.partitionSize ||
+      this.$scope.errorInst.partitionSizeBoot ||
+      this.$scope.errorInst.partitionSizeReiserfs ||
+      this.$scope.errorInst.partitionSizeWindows ||
+      this.$scope.errorInst.partitionSizeMin ||
+      this.$scope.errorInst.partitionSizeRequired
+    );
+  }
+
+  // boot size < 50Mo = error
+  validationSizeBoot(partition) {
+    this.$scope.errorInst.partitionSizeBoot =
+      partition.mountPoint === this.$scope.constants.warningBoot &&
+      this.getRealDisplaySize({
+        partition,
+        notDisplay: true,
+        noRaid: true,
+      }) < this.$scope.constants.minSizeBoot;
+    return this.$scope.errorInst.partitionSizeBoot;
+  }
+
+  // reiserfs size < 32Mo = error
+  validationSizeReiserfs(partition) {
+    this.$scope.errorInst.partitionSizeReiserfs =
+      partition.fileSystem === this.$scope.constants.warningReiserfs &&
+      this.getRealDisplaySize({
+        partition,
+        notDisplay: true,
+        noRaid: true,
+      }) < this.$scope.constants.minSizeReiserfs;
+    return this.$scope.errorInst.partitionSizeReiserfs;
+  }
+
+  // windows size < 20Go = error
+  validationSizeWindowsMin(partition) {
+    this.$scope.errorInst.partitionSizeWindows =
+      this.$scope.informations.softRaidOnlyMirroring &&
+      partition.mountPoint === this.$scope.constants.warningCwin &&
+      this.getRealDisplaySize({
+        partition,
+        notDisplay: true,
+        noRaid: true,
+      }) < this.$scope.constants.minSizeWindows;
+    return this.$scope.errorInst.partitionSizeWindows;
+  }
+
+  // partition size < 10Mo = error
+  validationSizeMin(partition) {
+    this.$scope.errorInst.partitionSizeMin =
+      this.getRealDisplaySize({
+        partition,
+        notDisplay: true,
+        noRaid: true,
+      }) < this.$scope.constants.minSizePartition;
+    return this.$scope.errorInst.partitionSizeMin;
+  }
+
+  validationSize(partition) {
+    if (partition.partitionSize) {
+      set(
+        partition,
+        'partitionSize',
+        parseInt(partition.partitionSize.toString().replace('.', ''), 10),
+      );
+    }
+    this.$scope.errorInst.partitionSizeRequired = !/^[0-9]{1,20}$/.test(
+      partition.partitionSize,
+    );
+
+    this.getRemainingSize();
+    return (
+      this.$scope.errorInst.partitionSizeOver ||
+      this.validationSizeBoot(partition) ||
```

Comment:
I'm not sure to understand. Because the condition is not same for the differentes function to validation size

Reply:
Each validation method is making this call:
this.getRealDisplaySize({
        partition,
        notDisplay: true,
        noRaid: true,
      })
It would be nice to pass the value directly to avoid calling it four times

Reply:
it's applied